packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1486,7 +1487,7 @@ abstract class ProxyWidget extends Widget {
   /// widget's child be a widget such as [Row], [Column], or [Stack], which have a
   /// `children` property, and then provide the children to that widget.
   /// {@endtemplate}
-  final Widget child;
+  final Widget? child;
```

Comment:
should be non-nullable

Reply:
Some null may be here accros all children.

Reply:
not sure what this means. ProxyWidget should never have a null child.

Reply:
There was `DefaultTextStyle.fallback` that was explicitly using null child. Please review https://github.com/flutter/flutter/pull/64672/commits/3750f394117651f6cb2593bbb5eb66112f3da708#diff-879a73856784f585a1665c1ace514899 and let me know if this looks good to you.

Reply:
yeah looks good